tests/e2e/cucumber/steps/ui/resources.ts

```diff
@@ -74,7 +74,8 @@ When(
     for (const { resource, to } of stepTable.hashes()) {
       await resourceObject[actionType === 'copies' ? 'copy' : 'move']({
         resource,
-        newLocation: to
+        newLocation: to,
+        method: method === null ? 'menu' : method
```

Comment:
```suggestion
        method: method ?? 'menu',
```

can you try it if it can work like this?